packages/config/src/discovery/values/hardcoded.ts

```diff
@@ -19,4 +19,13 @@ export const HARDCODED = {
     SET_VALIDATOR_COUNT: 7,
     SET_SEQUENCER_COUNT: 3,
   },
+  ZKSYNC_2: {
+    FACETS: [
+      '0xF1fB730b7f8E8391B27B91f8f791e10E4a53CEcc',
+      '0x6df4A6D71622860dcc64C1FD9645d9a5BE96f088',
+      '0x2a2d6010202B93E727b61a60dfC1d5CF2707c1CE',
+      '0xc796A402E1B26eCd2cd38F23e05A2f904504ec89',
+      '0x389a081BCf20e5803288183b929F08458F1d863D',
+    ],
```

Comment:
Why are those hardcoded?

Reply:
when any of the facets changes then the upgradability can change, so it should be re-evaluated